src/lcancel.c

```diff
@@ -204,6 +204,8 @@ void LCancel_Think(LCancelData *event_data, FighterData *hmn_data)
 
     JOBJ *hud_jobj = event_data->hud.gobj->hsd_object;
 
+    bool should_update_fastfall_hud = false;
```

Comment:
I'm not a C developer, so I don't know if this is the idiomatic way to do this. I just didn't feel like copy-pasting the ff chunk in two places. Please point me in the right direction!